### DISCUSSION TITLE
Front End Choice

### DISCUSSION THREAD

**Discussion by PizzaConsole** [Code2Gether-Discord/JokesOnYou#41] [Ideas]
I have really looked too much into the design of this yet. I am just curious on the reasoning behind choosing Angular as the front end? This is a group for C# programming. We could just as easily implement a Blazor front end, thus making the full app in C#... Thoughts?

**Comment by HellHunterMax** [comment]
If you want we can do them Side by Side some people know angular some know blazor might be great learning experience for both ?
If anyone is willing to do React or Vue that's fine too. don't know who will help so expect no help :P

**Reply by PizzaConsole** [reply]
Right thats what @chrisK00 and I discussed in the discord. I dont need any help but Daze expressed interesting in helping with the Blazor front end.
